### ISSUE TITLE
Adabins was trained in a fully supervised manner on the Nuscience dataset?

### ISSUE THREAD

**Issue by RuiTianHIT** [md4all/md4all#31]
@sgasperini @morbi25 
Hello author, thank you for your excellent work!
We have been following your high-quality work since last year and are very interested in it. I have a few technical questions that I would like to ask you.
(1) I saw in the paper that Adabins was trained in a fully supervised manner on the Nuscience dataset.
We have explored that the sparsity of the Nuscence depth estimation dataset is not conducive to direct use in supervised deep networks. For example, the following image shows the Nuscience dataset we trained in a fully supervised manner (the input image has not been any data processing and is only resized)
![crop](https://github.com/md4all/md4all/assets/104761701/88bcfff7-8e7b-4015-affd-7bdcaf225cea)

![n015-2018-08-02-17-16-37+0800__CAM_FRONT__1533201470412460](https://github.com/md4all/md4all/assets/104761701/cb773259-9908-4c96-b454-3a8a134303e5)

(2) Regarding Adabins training in a fully supervised manner on the Nuscence dataset, I would like to ask how you can handle the data due to the sparsity issue in collecting depth values. Thank you very much. If you could open source Adabins training in a fully supervised manner on the Nuscience dataset, I would greatly appreciate it. This issue has been troubling us for a long time.

Looking forward to your reply. Once again, thank you to the author!!!

**Comment by Junggy** [comment]
Hi, I am the one who trained AdaBins on that paper.
so, we also found out that issue when the network is tranied for longer epochs (although loss gets lower and lower in test set)

What we did was, we saved image per test epochs and selected one that was right before that artifact intensified. Note that earlier epoch may have worse test loss, but will have less artifacts like that

**Comment by RuiTianHIT** [comment]
@Junggy Thank you very much for your reply, we will follow your advice, thank you again! 
This issue has been resolved, I will close it later.

**Comment by sgasperini** [comment]
Hi @RuiTianHIT, you reopened the issue a week ago but did not specify why. Could you please comment on what is wrong or close the issue? Thank you

**Comment by RuiTianHIT** [comment]
@sgasperini 
Hello author!
Thank you very much!
I would like to ask further whether Adabins is trained on the Nusence dataset based on the fully supervised paradigm whether will cause the above-mentioned stripe visualization problem, or this is a normal phenomenon because there are very few pixels with depth involved in supervision.
Thank you again, I look forward to your reply!

**Comment by sgasperini** [comment]
Hi @RuiTianHIT, your question was already addressed above by @Junggy. To summarize, the stripes appear when training for too long and the model overfits on the little supervision available from the point cloud. As you see in our paper, the stripes were minimum or absent as we selected a checkpoint prior to the stripes formation.

**Comment by RuiTianHIT** [comment]
Thanks, I will close this question！
